ci-operator/config/openshift/cluster-version-operator/openshift-cluster-version-operator-master.yaml

```diff
@@ -68,6 +68,13 @@ tests:
   steps:
     cluster_profile: gcp
     workflow: openshift-upgrade-gcp-loki
+- as: e2e-rollback
+  steps:
+    cluster_profile: gcp
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:latest
+      OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE: release:initial
+    workflow: openshift-upgrade-gcp-loki
```

Comment:
Why make these distinct tests?  I'd rather avoid duplicate installs and set `abort-at=100` like we had before #12544 for all of these patch-level updates.

Reply:
`rollback` test is "install 4.x+1 and revert to 4.x", while "abort-at" is "install 4.x, update to 4.x+1 and rollback to 4.x". I think we want both? Not sure if "abort-at" covers more than proposed rollback test (also not sure how stable is it)

Reply:
hmm, seems initial install -> upgrade -> rollback would make more sense in CVO context. Reworked this to use "abort-at"